pkg/handler/handler.go

```diff
@@ -230,7 +237,11 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 					}
 
 					// serve the first matching media type in the sorted clause list
-					data, etag, lastModified := accepts.GetDataAndETag()
+					data, etag, lastModified, err := accepts.GetDataAndETag()
+					if err != nil {
+						w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Should we log the error so that we know what's going on from the logs if this fails?